scripts/asm-installer/install_asm

```diff
@@ -169,6 +171,9 @@ main() {
     if can_modify_gcp_components; then
       enable_workload_identity
       init_meshconfig
+      if [[ "${CA}" = "gcp_cas" ]]; then
+        init_gcp_cas
```

Comment:
Can we have a comment here saying that there's no associated check if you don't have --enable_gcp_components set, so it's possible to have a cluster configured to use privateca with no CSR privs

Reply:
Will the setup fail? Or will it install properly and not be able to request certs?

Reply:
the ASM installation will technically fail. The XDS will never be able to come up if the certificates are not obtained from the CA server...

Reply:
Ok, gtg as is then.